Frontend/ui-customer/src/Screens/OnBoardingFlow/EnterMobileNumberScreen/ScreenData.purs

```diff
@@ -15,15 +15,21 @@
 
 module Screens.EnterMobileNumberScreen.ScreenData where
 
-import Screens.Types (EnterMobileNumberScreenState)
+import Data.Monoid.MList (get)
+import Foreign.Object (empty)
 import MerchantConfig.DefaultConfig as DC
 import PrestoDOM (LetterSpacing(..))
-import Foreign.Object (empty)
+import Screens.Types (EnterMobileNumberScreenState)
 
 initData :: EnterMobileNumberScreenState
 initData = {
     data: {
       mobileNumber: ""
+    , countryObj: {
```

Comment:
Are we using this data?
If not please remove

Reply:
Yes wea are using